### PR TITLE
Adding generated JavaCC parser code.

### PR DESCRIPTION
**Pull request by Vizerai**

### CODE REVIEW

File: src/main/javacc/ASTFilename.java [hunk 1]
```diff
@@ -18,7 +19,7 @@ public String toString() {
     return "Filename: " + name;
   }
 
-  /** Accept the visitor. **/
+  /** Accept the visitor. * */
```
Comment (tinaspark) [inline src/main/javacc/ASTFilename.java:22]: clean up  extra space

File: src/main/javacc/copy.jjt [hunk 2]
```diff
@@ -10,15 +11,15 @@ package com.google.cloud.spanner.pgadapter.parsers.copy;
...
 public class Copy {
 
-  public static void parse(String stmt) throws Exception {
+  public static void parse(String stmt, CopyTreeParser.CopyOptions options) throws Exception {
     java.io.InputStream stmtStream = new java.io.ByteArrayInputStream(stmt.getBytes());
-    Copy t = new Copy(stmtStream);
+    Copy tree = new Copy(stmtStream);
     try {
-      ASTStart n = t.Start();
-      CopyVisitor v = new CopyDumpVisitor();
```
Comment (tinaspark) [inline src/main/javacc/copy.jjt:18]: is the CopyDumpVisitor still in use somewhere or should it be deleted in this PR?

**Review by tinaspark — CHANGES_REQUESTED** [review]
Can you update the description  of this PR to include what changed between the CopyTreeVisitor and the CopyTreeParser

**Comment by Vizerai** [comment]
I'm not sure why this merged.  I just merged another update to fix the tests and it auto-merged this one as well.  It looks like the new changes were not committed though.  There is nothing in the parsers/copy folder. I'll move the changes/updates to a new PR.

File: src/main/javacc/copy.jjt (hunk 2 above)
Comment (Vizerai) [reply src/main/javacc/copy.jjt:18]: The file was renamed multiple times: CopyDumpVisitor -> CopyTreeVisitor -> CopyTreeParser

File: src/main/javacc/ASTFilename.java (hunk 1 above)
Comment (Vizerai) [reply src/main/javacc/ASTFilename.java:22]: Done.

**Comment by Vizerai** [comment]
See https://github.com/GoogleCloudPlatform/pgadapter/pull/19

**Pull request merged**